src/encoder/encoderffmpegcore.cpp

```diff
@@ -194,30 +196,31 @@ void EncoderFfmpegCore::encodeBuffer(const CSAMPLE *samples, const int size) {
 //
 // Currently this method is used before init() once to save artist, title and album
 //
-void EncoderFfmpegCore::updateMetaData(char* artist, char* title, char* album) {
+void EncoderFfmpegCore::updateMetaData(const QString& artist, const QString& title, const QString& album) {
     qDebug() << "ffmpegencodercore: UpdateMetadata: !" << artist << " - " << title <<
              " - " << album;
     m_strMetaDataTitle = title;
```

Comment:
Lame is linked at runtime. In this case initEncoder() fails and a message Box is displayed. 
This is somhow a bad design to add a fallback encoder ...  

For this PR, it is probably the best to disable ffmpeg entirely. 

We have a nice fallback solution on the decoder side, but this is out of scope for this PR. 

This can be done simply by a local #ifdef. Please add a comment, that a contributor can pick it up.